integration-tests/src/test/java/oracle/kubernetes/operator/utils/Operator.java

```diff
@@ -428,6 +484,91 @@ private void initialize(String yamlFile) throws Exception {
     }
   }
 
+  private void initializeCustom(
```

Comment:
There is duplicate code in initializeCustom and initialize methods. Can that be pulled out into small methods or see if they can be combined? Otherwise LGTM.

Reply:
you are right, modified initialize() in this commit 
https://github.com/oracle/weblogic-kubernetes-operator/pull/924/commits/bfff51cb4f10b402e16f73c6df20e2868620bd97